python/cudaq/kernel/ast_bridge.py

```diff
@@ -2162,6 +2174,11 @@ def bodyBuilder(iterVal):
                         node)
 
                 if node.func.attr == 'qvector':
+                    if len(self.valueStack) == 0:
+                        self.emitFatalError(
+                            'qvector does not have nullary constructor. init from size or existing state.',
```

Comment:
nit: Wording of the error message, is 'default' constructor Pythonic than 'nullary' constructor?

Reply:
done, thanks!